src/main/java/org/jetbrains/research/intellijdeodorant/inheritance/TypeBindingInheritanceDetection.java

```diff
@@ -0,0 +1,85 @@
+package core.ast.inheritance;
+
+import java.util.*;
+
+public class TypeBindingInheritanceDetection {
+	// TODO: add me
```

Comment:
Seems that I accidentally copied this file from the original plugin. I also reviewed other files in the 'inheritance' package and removed 'InheritanceDetection.java' file because it was also not used (neither was it in the original project).